parsec/backend/postgresql/pki.py

```diff
@@ -200,6 +202,19 @@
     LIMIT 1
     """
 )
+_q_retrieve_active_human_by_email_for_update = Q(
+    f"""
+    SELECT
+        user_.user_id
+    FROM user_ LEFT JOIN human ON user_.human=human._id
+    WHERE
+        user_.organization = { q_organization_internal_id("$organization_id") }
+        AND human.email = $email
+        AND (user_.revoked_on IS NULL OR user_.revoked_on > $now)
```

Comment:
The `user_.revoked_on > $now` is not needed: we consider revoked any user that have `revoked_on` not NULL